_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,392 @@
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Terminal UI (TUI)
+
+@tarruda has merged his [pull request][tui-pr] to reimplement the terminal user
+interface using [libtermkey][tui-libtermkey] and [unibilium][tui-unibilium].
+
+Libtermkey is a library by @leonerd that "allows easy processing of keyboard
+entry from terminal-based programs." It has the ability to handle special keys,
+[UTF-8 combining][tui-combining] and more.
+
+Unibilium is a library by @mauke that interfaces with [terminfo][tui-terminfo].
+Terminfo is a library and database that makes it easy to interface with display
+terminals without worrying about device specific details. More details can be
+found on the man page for terminfo by running: [`man 5 terminfo`][tui-manpage].
+
+Although the TUI implementation is included as part of Neovim, it is
+architecturally decoupled from the editor core: it uses the same API as external
+clients and GUIs.
+
+#### Additional Keybindings
+
+As a result of using libtermkey, Neovim can now bind [meta keys][tui-meta] and
+`Ctrl-Shift` combinations both of which weren't previously possible in the
+terminal. Be sure to [read more][tui-bindings] about the updated binding
+capabilities for additional examples and possibilities.
+
+### Builtin Terminal Emulation
+
+@tarruda has finished [his pull request][term-pr] adding a new `:terminal`
+command which when ran launches a full terminal emulator. The terminal is using
+Neovim's native buffers and windows which means the user can enter [normal
+mode][term-normal] by pressing `<C-\><C-n>`.
+
+This is meant to replace and provide more capabilities than the already existing
+Vim command, `:shell`, which just launches a shell. More explanation on the
+decision of this feature can be found in [this comment][term-comment] made by
+@tarruda.
+
+The complete documentation for this feature can be found under [`:h
+nvim-terminal-emulator`][term-docs].
+
+This feature is implemented using another of @leonerd's projects called
+[libvterm][term-libvterm].
+
+### True Colors
+
+With the changes to the TUI described above, this allowed @tarruda to
+make a [pull request][color-pr] to add true colors to terminals that support it.
+
+A more detailed explanation of what true colors means as well as terminals that
+support it can be found in [this Gist][color-info] that was written by @XVilka.
+
+To enable true colors, set the environment variable
```

Comment:
An exported environment variable is necessarily set, but an environment variable is not necessarily exported, so I think you're right.

One thing though: the way it's worded might make people think that the following is enough:
```
export NVIM_TUI_ENABLE_TRUE_COLOR
nvim
```

when in reality, the first results in nothing being exported (at least according to `env` in both `bash` and `mksh`). Given that the corresponding code just checks if the environment variable exists, then maybe the sentence being discussed could say something like:
```
... set the environment variable `NVIM_YAYA` to `1`, e.g.

export NVIM_YEYE=1
nvim

or just launch nvim like this (assuming you're using a bourne/korn shell derivative):

NVIM_YEYE=1 nvim
```

It might just be better to assume people aren't using `tcsh` or `csh`.